### DISCUSSION TITLE
java-features/java17/jep406-pattern-matching-for-switch-preview

### DISCUSSION THREAD

**Discussion by giscus** [dyc87112/giscus-didispace#20] [Announcements]
# java-features/java17/jep406-pattern-matching-for-switch-preview

Java 17 新特性：switch的模式匹配（Preview） 还记得Java 16中的instanceof增强吗？ 通过下面这个例子再回忆一下： Map<String, Object> data = new HashMap<>(); data.put(

https://www.didispace.com/java-features/java17/jep406-pattern-matching-for-switch-preview.html

**Comment by fanxiangqing** [comment]
111

**Reply by fanxiangqing** [reply]
1

**Comment by fanxiangqing** [comment]
1
